src/state/pools/index.ts

```diff
@@ -150,8 +151,12 @@ export const fetchPoolsPublicDataAsync = (currentBlockNumber: number) => async (
     const liveData = poolsConfig.map((pool) => {
       const blockLimit = blockLimits.find((entry) => entry.sousId === pool.sousId)
       const totalStaking = totalStakings.find((entry) => entry.sousId === pool.sousId)
+      const isPoolStartTooFar =
+        currentBlock > 0 && blockLimit
+          ? Number(blockLimit.startBlock) > currentBlock + (60 / BSC_BLOCK_TIME) * 4
+          : false
       const isPoolEndBlockExceeded = currentBlock > 0 && blockLimit ? currentBlock > Number(blockLimit.endBlock) : false
-      const isPoolFinished = pool.isFinished || isPoolEndBlockExceeded
+      const isPoolFinished = pool.isFinished || isPoolEndBlockExceeded || isPoolStartTooFar
```

Comment:
If we use isPoolFinished, this will put that new pool to the finished pools. I think we need to create a new property.

Reply:
Imho this should be handled in view side

Reply:
1. change `isPoolFinished` name: semantic is better. but how about `isFinishded`, all pool config has static isFinished flag.
2. view, if we want to reduce re-render, have to use useInitialBlock instead of useCurrentBlock in view, that will work, just need to refresh in order to see new pools.

Reply:
But I think with this approach the pool with far start block will be shown in finished pools, if we use that param to check.